common/wireaddr.h

```diff
@@ -22,18 +23,23 @@
  *             where `checksum = sha3(".onion checksum" | pubkey || version)[:2]`
  */
 
+#define	TOR_V2_ADDRLEN 12
+#define	TOR_V3_ADDRLEN 37
```

Comment:
`TOR_V2_ADDRLEN` should be `10` and `TOR_V3_ADDRLEN`  should be `35`; last 2 bytes are for the port, which is the next field in `struct wireaddr`

Reply:
Just noticed that @rustyrussell has fixed the bug in his branch 👍